deploy/examples/psp.yaml

```diff
@@ -0,0 +1,259 @@
+####################################################################################################
```

Comment:
How about creating the psp.yaml in one of the integration tests that runs on an older k8s version to give some basic validation that it can be created?

Reply:
I added a conditional to the github action helper `validate_yaml()` function.